system/Database/SQLSRV/Forge.php

```diff
@@ -144,7 +145,25 @@ protected function _alterTable(string $alterType, string $table, $field)
                 }
             }
 
-            $sql = 'ALTER TABLE ' . $this->db->escapeIdentifiers($this->db->schema) . '.' . $this->db->escapeIdentifiers($table) . ' DROP ';
```

Comment:
This is from my other PR: https://github.com/codeigniter4/CodeIgniter4/pull/6430
This allows all my tests to run.